docs/approvals/index.md

```diff
@@ -5,7 +5,10 @@ position: 95
 hideInThisSection: true
```

Comment:
If you remove the section header, and not the entire contents, the page becomes more dynamic
```suggestion
hideInThisSectionHeader: true
```